32blit-sdl/Main.cpp

```diff
@@ -124,6 +124,9 @@ void em_loop() {
 }
 #endif
 
+#ifdef __cplusplus
+extern "C"
+#endif
```

Comment:
I think you can avoid this one by linking to SDL2main.

Reply:
Whoops, that was left there by accident, good catch. Thx! :)